photon-core/src/main/java/org/photonvision/raspi/PicamJNI.java

```diff
@@ -86,6 +88,7 @@ public static synchronized void forceLoad() throws IOException {
 
     public static boolean isSupported() {
         return libraryLoaded
+                && enabled
```

Comment:
Can we do something a bit more intelligent here?

Reply:
No. Disabling the driver until after release.